src/pages/trade/HorizontalPanel.tsx

```
@@ -181,61 +181,61 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen, handleStartResize }:
   );
 
   const ordersTabItem = useMemo(
```

ESLint

[error] 'ordersTabItem' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)